api/v1beta1/clusterrelocation_types.go

```diff
@@ -121,8 +121,15 @@ type RegistryCert struct {
 }
 
 const (
-	ConditionTypeReady string = "Ready"
-	ConditionTypeApi   string = "APIReconciled"
+	ConditionTypeReady      string = "Ready"
+	ConditionTypeApi        string = "APIReconciled"
+	ConditionTypePullSecret string = "PullSecretReconciled"
+)
+
```

Comment:
why multiple const are defined instead just one with all the constants?

Reply:
I guess it's just an organization thing, to organize the different types of constants into groups, it's how I see other projects do it, for instance: https://github.com/openshift/assisted-service/blob/8bd54379675a9fabce065d69433e4e07dcc7cbac/api/hiveextension/v1beta1/agentclusterinstall_types.go#L362-L390